code/modules/materials/Mat_Materials.dm

```diff
@@ -1156,8 +1186,12 @@ ABSTRACT_TYPE(/datum/material/crystal)
 	mat_id = "telecrystal"
 	name = "telecrystal"
 	desc = "Telecrystal is a gemstone with space-warping properties."
-	color = "#4C14F5"
-	alpha = 100
+	color = list(0.50, 0.25, 0.25, 0.00,\
+				0.25, 0.00, 0.25, 0.00,\
+				0.35, 0.25, 0.45, 0.00,\
+				0.00, 0.00, 0.00, 0.75,\
+				0.00, 0.00, 0.00, 0.00)
+	alpha = 191
```

Comment:
These alpha values are actually meaningful to material behaviour (see through windows, captain's antique, glasses etc.) and shouldn't be changed without good reason